packages/tools/devtools/devtools-view/src/components/data-visualization/FluidHandleView.tsx

```diff
@@ -87,5 +88,27 @@ export function FluidHandleView(props: FluidHandleViewProps): React.ReactElement
 		return <TreeItem header={header} />;
 	}
 
-	return <TreeDataView containerKey={containerKey} label={label} node={visualTree} />;
+	//Button to inlcude in header to copy fluid handle
+	const button = (
+		<Tooltip content={`Click to copy handle`} relationship={"label"}>
+			<Button
+				onClick={async (e): Promise<void> => {
+					e.preventDefault();
+					return navigator.clipboard.writeText(fluidObjectId);
+				}}
+				icon={<ClipboardPaste16Regular />}
+				size="small"
+			></Button>
+		</Tooltip>
+	);
+
+	const header2 = (
```

Comment:
This is the only header here, so I think we can just call it "header".

Reply:
If visualTree is undefined than it defines its own header. If I change header2 to just header it compalins about it being "already defined in this scope" since its a const (I think)

Reply:
If visualTree is undefined than it defines its own header. If I change header2 to just header it compalins about it being "already defined in this scope" since it's a const (I think). What are your thoughts? I imagine some other name makes sense but I'm not sure.

Reply:
Oh, I completely missed that. That might be a good motivation to wrap the remainder of the method in an `else` block (so we can use the same variable name).

Reply:
I think this is resolved.